index.html

```diff
@@ -38,15 +38,14 @@
                 <p class="logo">Architects</p>
                 <ul class="nav-list flex">
                     <li><a href="/about.html">About</a></li>
-                    <li><a href="">Prizes</a></li>
-                    <li><a href="">Biography</a></li>
-                    <li><a href="">Projects</a></li>
+                    <li><a href="#champions">Facts</a></li>
+                    <li><a href="#architect">Speakers</a></li>
```

Comment:
- [ ] Please follow the design provided to implement some features, Compare the two images:

--------------------------------------Original-----------------------------------------------
![nav1](https://user-images.githubusercontent.com/55635977/128219460-c76b94d7-a3c2-43f6-8217-4ca8bf21f1ec.png)

--------------------------------------Yoursl-----------------------------------------------
![nav2](https://user-images.githubusercontent.com/55635977/128219501-bb662b3a-0022-480b-988f-37d8af82558a.png)

Reply:
- This issue is not properly resolved 😿